packages/devtools_app/test/flutter/memory_screen_test.dart

```diff
@@ -79,7 +79,7 @@ void main() {
 
       expect(controller.memorySource, MemoryController.liveFeed);
 
-      expect(find.byKey(MemoryScreen.snapshotButtonKey), findsOneWidget);
+//      expect(find.byKey(MemoryScreen.snapshotButtonKey), findsOneWidget);
```

Comment:
why is this commented out?

would be good to add some more tests as a followup. now that you are using a snapshot json file, writing unittests without requiring a vm service will be a lot easier.